src/api/spec/models/comment_spec.rb

```diff
@@ -2,9 +2,9 @@
 
 RSpec.describe Comment do
   let(:comment_package) { create(:comment_package) }
-  let(:comment_package_with_parent) { create(:comment_package, parent: comment_package) }
-  let(:comment_package_with_parent_2) { create(:comment_package, parent: comment_package) }
-  let(:comment_package_with_grandparent) { create(:comment_package, parent: comment_package_with_parent) }
+  let(:comment_package_with_parent) { create(:comment_package, parent: comment_package, commentable: comment_package.commentable) }
```

Comment:
It is needed for the comment model to validate the parent_id properly during the test.